src/pages/getting-started/installing-livechat/index.mdx

```diff
@@ -11,12 +11,12 @@ desc: "Installing LiveChat is a breeze and takes only a few minutes. Follow the
 
 Installing LiveChat is simple and only takes a few minutes. Once installed, you’ll be able to talk to your customer’s right away.
 
-1. Log in to your LiveChat account and go to Settings &rarr; Channels &rarr; [Chat widget](https://my.livechatinc.com/settings/code).
-2. Then, click on Copy code.
+1. Log in to your LiveChat account and go to Settings &rarr; Channels &rarr; [Website](https://my.livechatinc.com/settings/code).
+2. Click on Install LiveChat code manually and then Copy code.
```

Comment:
Perhaps we could use bold for the button copy?

Reply:
good idea

Reply:
@wfabjanczuk I have given suggestions for words that should be bold :)